src/test/python/test_VantiqSDKLive.py

```diff
@@ -935,6 +957,74 @@ async def test_subscriptions_as_plain_client(self):
         await self.check_subscription_ops(client, False)
         await client.close()
 
+    @pytest.mark.asyncio
+    @pytest.mark.timeout(10)
+    async def test_subscriptions_as_plain_client_ignore_ssl(self):
+        self.check_test_conditions()
+        client = Vantiq(_server_url, '1', ssl=False)
+        if _access_token:
+            await client.set_access_token(_access_token)
+        else:
+            await client.authenticate(_username, _password)
+        await self.check_subscription_ops(client, False)
+        await client.close()
+
+    @pytest.mark.asyncio
+    @pytest.mark.timeout(10)
+    async def test_subscriptions_as_plain_client_ignore_ssl_no_version(self):
+        self.check_test_conditions()
+        client = Vantiq(_server_url, ssl=False)
+        if _access_token:
+            await client.set_access_token(_access_token)
+        else:
+            await client.authenticate(_username, _password)
+        await self.check_subscription_ops(client, False)
+        await client.close()
+
+    @pytest.mark.asyncio
+    @pytest.mark.timeout(10)
+    async def test_subscriptions_as_plain_client_ssl_context(self):
+        self.check_test_conditions()
+        context = ssl.create_default_context()
+        context.check_hostname = False
+        context.verify_mode = ssl.CERT_NONE
+        client = Vantiq(_server_url, '1', ssl=context)
+        if _access_token:
+            await client.set_access_token(_access_token)
+        else:
+            await client.authenticate(_username, _password)
+        await self.check_subscription_ops(client, False)
+        await client.close()
+
+    @pytest.mark.asyncio
+    @pytest.mark.timeout(10)
+    async def test_subscriptions_as_plain_client_should_reject(self):
+        self.check_test_conditions()
+        if not _server_url.startswith('https://'):
+            pytest.skip('Server URL must be SSL for this test')
+        # Here, we create an "expected" fingerprint that shouldn't match much any real certificate.
+        # We expect this to fail. The test verifies that any attempt to do any work
+        # will fail because of the SSL context (with the fingerprint) that we sent in.  This positively
+        # verifies that our contexts are being passed all the way through/
+        context = aiohttp.Fingerprint(b'0' * 32)
+        exc = None
+        client = Vantiq(_server_url, '1', ssl=context)
```

Comment:
Given a hokey fingerprint, any connection should fail.  Verify that that's the case.